MarvelApiClient/CharactersAPIClient.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  CharactersAPIClient.swift
+//  MarvelAPIClient
+//
+//  Created by Pedro Vicente on 14/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamNetworking
+import Result
+
+public class CharactersAPIClient {
+
+    private let apiClient: BothamAPIClient
+    private let parser: CharactersParser
+
+    init(apiClient: BothamAPIClient, parser: CharactersParser) {
+        self.parser = parser
+        self.apiClient = apiClient
+    }
+
+    public func getAll(offset offset: Int, limit: Int,
+        completion: (Result<GetCharactersDTO, BothamAPIClientError>) -> ()) {
+        assert(offset >= 0 && limit >= 0)
+        let params: [String:String?] =  [MarvelAPIParams.offset : "\(offset)", MarvelAPIParams.limit : "\(limit)"]
```

Comment:
I'm fine if we change in the library if we consider as default behaviour, and we open a ticket to implement as a config, that:
// By Default
["var1" : ""] generate var1=
// and set 
["var1" : ""] generate var1